search-api.md

```diff
@@ -230,3 +224,26 @@ A synchronous `application/json` response, of the following form:
 #### Results
 * ***Mandatory***, but can be empty
 * Is a **list of matches**, where each match has a `patient` object of the same format as the one described above for the query
+
+
+### Error handling
+The remote server should use HTTP status codes to report any errors encoundered processing the match request. Here are a list of status codes and their meanings with regards to this API:
+
+| HTTP Status Code | Constant | Description
+| ---------------- | -------- | -----------
+| 200 | httplib.OK | no error |
```

Comment:
can probably replace by names as in http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html (which are basically the same without the prefix)